docs/_data/sidebars/left-navigation-sidebar.yml

```diff
@@ -4,26 +4,10 @@ entries:
 
   folders:
 
-  # foundation
   - title: getting started
     url: /getting-started.html
     output: web
 
-  # foundation
-  - title: foundation
-    url: /foundation/index.html
-    output: web
-
-    folderitems:
-
-    - title: Spacing
-      url: /foundation/spacing.html
-      output: web
-
-    - title: Type
-      url: /foundation/type.html
-      output: web
-
```

Comment:
Removed the foundation pages as they are no longer correct (and haven't been for a while)